packages/data-ingestion/lib/index.ts

```diff
@@ -0,0 +1,104 @@
+import { BigQuery } from '@google-cloud/bigquery';
+import type { BigQuery as BigQueryType } from '@google-cloud/bigquery';
+import { getLogger } from '@nangohq/utils/dist/logger.js';
+import { isCloud, isLocal } from '@nangohq/utils/dist/environment/detection.js';
+
+const logger = getLogger('BigQueryClient');
+
+interface RunScriptRow {
+    executionType: string;
+    internalConnectionId: number | undefined;
+    connectionId: string;
+    accountId: number | undefined;
+    scriptName: string;
+    scriptType: string;
+    environmentId: number;
+    providerConfigKey: string;
+    status: string;
+    syncId: string;
+    content: string;
+    runTimeInSeconds: number;
+    createdAt: number;
+}
+
+class BigQueryClient {
+    private client: BigQuery;
+    private datasetName: string;
+    private tableName: string;
+
+    constructor({ datasetName, tableName }: { datasetName: string; tableName: string }) {
+        this.client = new BigQuery();
+        this.tableName = tableName;
+        this.datasetName = datasetName;
```

Comment:
Might be interesting to throw on isCloud if the env var is empty

Reply:
yeah, will leave this out though for now